### DISCUSSION TITLE
Icon Suggestions

### DISCUSSION THREAD

**Discussion by not-a-dev-stein** [Heroic-Games-Launcher/HeroicGamesLauncher#187] [Ideas]
With the project now having made a name for itself (and gaining more and more traction) I think a new logo could come in handy, or at least an updated tray icon. The logo we have until now is okay, but having no transparency in the background makes it stick out a lot in there.

So I took a little time to brainstorm an updated version without changing too much of the original idea, and [here it is](https://imgur.com/jRCW9t7).

And also created a light-dark variant for the system tray:
[Light version](https://imgur.com/E8zAKii);
[Dark version](https://imgur.com/aic4Ki7).

This is totally not ready to use material, but more of an idea of how, if redesigned, how it should look. Chose colors similar to the Epic Games Store logo in order to sell the similarities of the launcher, while we don't get other launchers integrated with it.

If needed for something I can also upload the original .svg files for quick editing.

**Comment by flavioislima** [comment]
Good! Will be live on next release!

**Comment by flavioislima** [comment]
Just released new version, check how are the icons. If you use plasma, you will need to change the GTK theme for it to work.

**Comment by not-a-dev-stein** [comment]
Just downloaded it, didn't even need to change themes for it to work. Looks nice, but I think using .svg files instead of .png ones will scale better with the tray's size.

Uploaded them just in case:
https://drive.google.com/drive/folders/11pEAnRcCKya67lwrnDe7iJVWPZtJwyPD?usp=sharing

**Comment by flavioislima** [comment]
I requested access.
One thing I needed to do was to resize the image because using the original size was giving me too much aliasing. A SVG might be better.

**Comment by flavioislima** [comment]
So, SVG is not supported for tray icons. So I did convert the SVG to PNG and I think looks better now. You will be able to check on next version

**Comment by not-a-dev-stein** [comment]
> So, SVG is not supported for tray icons.

Oh, I didn't know that. Glad it at least was useful, hope it ends up looking good :)

**Comment by jarrard** [comment]
was going to point this out sometime. I use a dark desktop and the icon is dark, doesn't really work :)

Tried /home/theriddick/.config/gtk-3.0/settings.ini
Materia
Materia-dark
breeze-dark
breath-dark

Tray Icon is still dark... :(

Often this sort of oddness is a plasma icon auto shading thing.

**Comment by flavioislima** [comment]
This is me using Breath-Dark on plasma:
![image](https://user-images.githubusercontent.com/26871415/106194483-3158b400-61af-11eb-8346-d8f8e1360465.png)

**Comment by not-a-dev-stein** [comment]
Same here with Papirus, both regular and dark, works just fine.
![Screenshot_20201002](https://user-images.githubusercontent.com/63420929/106194840-19bd0380-618e-11eb-935b-2a5d5664d2af.png)

**Comment by jarrard** [comment]
I use Manjaro-Materia-Dark theme

![Screenshot_20210129_115233](https://user-images.githubusercontent.com/17428910/106219053-93093480-6228-11eb-9162-f8e84de822fd.png)

**Comment by not-a-dev-stein** [comment]
Had some time this week to work on some visual improvements on the logo.

So I ended up with these:
![heroic-light](https://user-images.githubusercontent.com/63420929/106940495-5f367f00-6700-11eb-9e26-52f75946eb49.png)
![heroic-dark](https://user-images.githubusercontent.com/63420929/106940506-63629c80-6700-11eb-977e-e316281bf65e.png)

Decided to close the lines since it wasn't clearly visible in the tray and went with thicker lines and more consistent spacing to make it clearer in the system tray. Even tested it using the file manager itself to compare:
![Screenshot_20210204_154203](https://user-images.githubusercontent.com/63420929/106940594-81300180-6700-11eb-84ad-cfec1c323a8f.png)

It's still not perfect, but at least I see it as a big improvement in visibility.

**Comment by flavioislima** [comment]
Nice, I was about to ask you to 'close' the lines as well. Really good. I'll change for the next release this weekend.

**Comment by jarrard** [comment]
The black one is what I get in dark theme, can't see it until I hover over icon and it turns to the lighter color version. So I dunno why its picking wrong shade. No other apps have a issue so far.

**Comment by ajhbh** [comment]
![dark-logo-cinnamon-dark](https://user-images.githubusercontent.com/35773116/107161456-c5d3cb00-6994-11eb-9fc0-6c5ed74578a6.png)
![dark-logo-cinnamon-dark-hover](https://user-images.githubusercontent.com/35773116/107161458-c8cebb80-6994-11eb-80bd-4665afe9b4e7.png)
Firstly nice work on the logos and Heroic generally! But I am having the same problem as jarrard where the logo is currently invisible in the system tray unless hovered over.
I'm using Cinnamon desktop with Mint-Y-Dark theme.

**Comment by jarrard** [comment]
I've just edited the images myself to have a base background so it shows up.

**Comment by flavioislima** [comment]
@not-a-dev-stein maybe having a base color background might be a good idea to solve these edge cases.
I forgot to add the new icons on the new release, sorry for that, but the translation took me a lot of time to finish.
But I can put them on a hotfix but would be good to test with different environments first.

To manually test, you just need to replace the icons on this folder:
`/opt/heroic/resources/app.asar.unpacked/build/`

And reopen Heroic.

**Comment by ajhbh** [comment]
![heroic-icons-switchNames](https://user-images.githubusercontent.com/35773116/107217297-e425df00-6a05-11eb-92eb-1f203e4ef1fa.png)
Cool I have tried that on my system and found for me it uses the 'right' colour icon when I swap the names around from what they were by default, i.e. naming the black icon icon-dark and the white one icon-light

**Comment by flavioislima** [comment]
> ![heroic-icons-switchNames](https://user-images.githubusercontent.com/35773116/107217297-e425df00-6a05-11eb-92eb-1f203e4ef1fa.png)
> Cool I have tried that on my system and found for me it uses the 'right' colour icon when I swap the names around from what they were by default, i.e. naming the black icon icon-dark and the white one icon-light

Thats one of the amazing things of Linux. You have a white theme but the bar is dark. 

I think the best way to solve this is to add an option on Heroic to choose the color of the icon. Automatically it will never work on all desktops with the amount of different themes that we have.

**Comment by flavioislima** [comment]
Hey guys! I'll convert this into a discussion for further improvements on the icons.

**Comment by Suzie97** [comment]
Remove the blue box on the colored icon. I think the blue box is not necessary and makes the icon big for no reason. It doesn't look all too good with it either.

The symbolic icons look amazing and I think if the colored icon would be a bit more refined to look more modern, the three icons would complement each other well.

**Comment by atraest** [comment]
Hello. I've made 3 mockup proposals trying to improving the logo aesthetics and readability.

![Frame 1](https://user-images.githubusercontent.com/57003922/228872363-092dbec5-3669-4f87-8da2-3a582d6bb89d.png)
